gopro2gpx/config.py

```diff
@@ -13,68 +13,38 @@
 import subprocess
 import re
 
-class E:
-    def __init__(self):
-        pass
-
 class Config:
-    def __init__(self, ffmpeg, ffprobe):
-        self.ffmpeg_cmd = ffmpeg
-        self.ffprobe_cmd = ffprobe
-        # get ffmpeg version
-        self.version = E()
-        self.version.major = 4
-        self.version.medium = 3
-        self.version.minor = 1
-        self.getFfmpegVersion()
-        self.use_json_format = False
-        if self.version.medium >= 4:
```

Comment:
I'm guessing this (`medium`) was a bug. I've interpreted the logic to be all **major** versions of ffprobe above 4 supported json output. Let me know if this assumption is incorrect.

Reply:
You are right, medium should be 4 instead of 3.

Reply:
What I mean is that the conditional checks the `self.version.medium` field:
```
if self.version.medium >= 4:
```
That is the, middle field of the semver spec. I'm guessing this should be querying the `self.version.major` field and updated the logic as such in `ffmpegtools.py` (I'll add a note below).